site/en/docs/extensions/mv3/match_patterns/index.md

```diff
@@ -3,154 +3,66 @@ layout: 'layouts/doc-post.njk'
 title: 'Match patterns'
 seoTitle: 'Chrome Extensions match patterns'
 date: 2012-09-18
-updated: 2023-07-28
+updated: 2023-08-31
 description: Understanding URL match patterns in Chrome extensions.
 ---
 
-## Overview {: #overview }
-
-Extensions use match patterns to specify a group of URLs. They are mainly used to declare [host permissions][host-permissions] along with other permissions to use certain Chrome APIs or when injecting [content scripts][content-scripts]. Extensions also use match patterns in:
-
-- [`"web_accessible_resources.matches"`][war]
-- [`"externally_connectable.matches"`][ext-connect]
-
-## What is a match pattern? {: #what }
-
-A match pattern is a URL that begins with a permitted scheme, separated by a `://`, followed by a host and path. It can contain wildcard (`*`) characters. Most match patterns are composed of three parts:
+A match pattern is a URL with the following structure, used to specify a group of URLs:
 
 ```text
 &lt;scheme&gt;://&lt;host&gt;/&lt;path&gt;
 ```
 
-Each part can use wildcards `*`. Below is a detailed description:
-
-- **scheme**: Must include a valid scheme: `'http'`, `'https'`, `'file'`, `'ftp'`, or `'urn'`. The wildcard `*` only matches `http` or `https`.
-
+- **scheme**: Must be one of `'http'`, `'https'`, `'file'`, `'ftp'`, or `'urn'`. The wildcard `*` only matches `http` or `https`.
+  - For information on injecting content scripts into unsupported schemes, such as `about:` and `data:`, see [Injecting in related frames][cs-frames].
+  - The scheme must be separated from the rest of the pattern using a double slash (`//`).
 - **host**: A hostname (`www.example.com`), a `*` before the hostname to match subdomains (`*.example.com`), or just a wildcard `*`.
+  - If you use a wildcard in the host pattern, it must be the first or only character, and it must be followed by a period (`.`) or forward slash (`/`).
+- **path**: Must contain at least a forward slash. The slash by itself is treated as a wildcard `/*`. Other examples include the following:
```

Comment:
@AmySteam do you know what the `/*` is trying to tell me?

Reply:
@AmySteam

Reply:
> '/' means the same thing as '/*', but do we really need to tell people that? It's what linguist Bryan Garner would call a [needless variant](https://www.cjr.org/analysis/there_ought_to_be_alternative.php#:~:text=might%20call%20a%20%E2%80%9C-,needless%20variant,-%E2%80%9D%3A%20%E2%80%9Ctwo%20or%20more)

It's just more explicit for developers to know if you add `*.example.com/`, it will also match `*.example.com/foo` and `*.example.com/bar`, not just `www.example.com`.

I think when it comes to match patterns, it's okay to be explicit.  But at this point I'm ready to agree with anything ya'll decide.

Reply:
Is this phrasing better?

Reply:
![image](https://github.com/GoogleChrome/developer.chrome.com/assets/37001393/7eaee830-4b7a-4b24-a9a1-2ff473c5084a)